src/picterra/client.py

```diff
@@ -878,7 +883,11 @@ def upload_vector_layer(
         upload = resp.json()
         upload_id, upload_url = upload["upload_id"], upload["upload_url"]
         _upload_file_to_blobstore(upload_url, filename)
-        data = {"name": name} if name else None
+        data = {}
```

Comment:
This changes the default value of data from `None` to and empty dict `{}`, I'm guessing without testing that the end result is the same (i.e. posting no data or an empty dict to the API has the same effect?) Just wanted to check!

Reply:
yes, good point: just checked, and it has the same effect